libs/checkpoint-validation/src/spec/get_tuple.ts

```diff
@@ -0,0 +1,249 @@
+import {
+  CheckpointTuple,
+  PendingWrite,
+  TASKS,
+  uuid6,
+  type BaseCheckpointSaver,
+} from "@langchain/langgraph-checkpoint";
+import { CheckpointerTestInitializer } from "../types.js";
+import {
+  parentAndChildCheckpointTuplesWithWrites,
+  putTuples,
+} from "../test_utils.js";
+
+export function getTupleTests<T extends BaseCheckpointSaver>(
+  initializer: CheckpointerTestInitializer<T>
+) {
+  describe(`${initializer.checkpointerName}#getTuple`, () => {
+    let checkpointer: T;
+    beforeAll(async () => {
+      checkpointer = await initializer.createCheckpointer();
+    });
+
+    afterAll(async () => {
+      await initializer.destroyCheckpointer?.(checkpointer);
+    });
+
+    describe.each(["root", "child"])("namespace: %s", (namespace) => {
+      let thread_id: string;
+      const checkpoint_ns = namespace === "root" ? "" : namespace;
+
+      let parentCheckpointId: string;
+      let childCheckpointId: string;
+
+      let generatedParentTuple: CheckpointTuple;
+      let generatedChildTuple: CheckpointTuple;
+
+      let parentTuple: CheckpointTuple | undefined;
+      let childTuple: CheckpointTuple | undefined;
+      let latestTuple: CheckpointTuple | undefined;
+
+      beforeAll(async () => {
+        thread_id = uuid6(-3);
+        parentCheckpointId = uuid6(-3);
+        childCheckpointId = uuid6(-3);
+
+        const writesToParent = [
+          {
+            taskId: "pending_sends_task",
```

Comment:
I think the library always uses uuids for taskId, are we using something else here on purpose to test it supports arbitrary strings?